api/BasicCardRequest.json

```diff
@@ -0,0 +1,154 @@
+{
+  "api": {
+    "BasicCardRequest": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/BasicCardRequest",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": "57"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "56",
+            "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
```

Comment:
Use the "flags" property instead of a "notes" entry.

Reply:
Like this?
```
"firefox": {
            "version_added": "56",
            "flags": [
              {
                "type": "preference",
                "name": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
              }
            ]
          },
```

Reply:
Almost, for "name" just "dom.payments.request.enabled"; followe by "value_to_set": "true"

See https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags